04-Enumerable/Enumerable/Task.cs

```diff
@@ -140,9 +159,17 @@ public class Task {
         ///   { "", null } => { }
         ///   { } => { }
         /// </example>
-        public IEnumerable<char> GetUsedChars(IEnumerable<string> data) {
+        public IEnumerable<char> GetUsedChars(IEnumerable<string> data)
+        {
             // TODO : Implement GetUsedChars
-            throw new NotImplementedException();
+
+            string s = "";
+            foreach (var item in data)
+            {
+                s += item;
+            }
+            return s.ToCharArray().Distinct(); ;
```

Comment:
Please think how to do it without foreach and not create a lot of additional strings